hscontrol/policy/v2/types.go

```diff
@@ -382,16 +382,21 @@ func (p Prefix) Resolve(_ *Policy, _ types.Users, nodes types.Nodes) (*netipx.IP
 type AutoGroup string
 
 const (
+	AutoGroupMember   AutoGroup = "autogroup:member"
 	AutoGroupInternet AutoGroup = "autogroup:internet"
+	AutoGroupTagged   AutoGroup = "autogroup:tagged"
```

Comment:
nit: we can sort this list alphabetically

Reply:
done in https://github.com/juanfont/headscale/commit/2b3ee902a711b9c5936ce6021c405c1fcb08103b